{"name": "测试用例描述：错误的账号登录失败", "status": "passed", "description": "测试描述：错误的账号登录失败", "steps": [{"name": "输入错误的账号", "status": "passed", "attachments": [{"name": "错误的账号", "source": "a0cc4e13-6266-4e38-bfb7-e36ac402c517-attachment.attach"}], "start": 1658301807544, "stop": 1658301807544}], "attachments": [{"name": "截图", "source": "4fa4a5cd-14e6-44b7-9d10-961986d01500-attachment.png", "type": "image/png"}], "start": 1658301807544, "stop": 1658301811206, "uuid": "b7ce84c8-2984-425d-b824-b6d228ba07e7", "historyId": "dac4ae518a5085ee93a0eeeab00b80d4", "testCaseId": "60e0d8659bd2e52aa6ebe8c64f9be9b0", "fullName": "test_login.Test#test_21", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "登录模块"}, {"name": "story", "value": "登录"}, {"name": "epic", "value": "xxx测试"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "4472-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}